### ISSUE TITLE
Create an SAPUI5 App from the Command Line : creation of MTA archive in error

### ISSUE THREAD

**Issue by chaouki-akir** [SAPDocuments/Tutorials#4490]
Tutorial URL: [https://developers.sap.com/tutorials/cp-cf-sapui5-local.html](https://developers.sap.com/tutorials/cp-cf-sapui5-local.html)

Hello,

I am following the steps of the above tutorial
Step 1-OK-Scaffold a SAPUI5 project (yo easy-ui5)
Step 2-OK-Inspect the generated code (cd com.myorg.myUI5App/ code .)
Step 3-OK-Test the web-app locally (npm start)
Step 4-KO-Package the MTA archive (npm run build:mta)

There is an error in the step 4 when I execute instruction npm run build:mta
![image](https://user-images.githubusercontent.com/23510132/72482082-caaf6500-37fc-11ea-90bb-744ab5764e7c.png)

Related issue #4155 4155

**Comment by IObert** [comment]
Thanks for opening this issue.

It seems like you've done everything right. The only thing that looks a bit strange, is that you created the UI5 app in the `system32` folder. Maybe there are some permission errors coming into play here.

Can you please try the following things:
1. Use the command `cd` to [switch into your home directory](https://sid-500.com/2017/08/03/powershell-how-to-quickly-switch-to-your-home-directory/) and please start over in this directory.
2. Does the same error occur when you use the Windows command line instead of PowerShell?
3. Can you update `mbt` in your project by running `npm i mbt` after step 2?

These are just guesses at the moment, please let me know if any of these work

**Comment by chaouki-akir** [comment]
1.
I switch into my home directory with PowerShell.
I tried the Step 1 yo easy-ui5 and it fails :
![image](https://user-images.githubusercontent.com/23510132/72572639-70c5a280-38c3-11ea-8a4e-c81cb1db58ff.png)

2.
I switch into my home directory with Windows command line.
I start over and the Steps 1 to 4 are OK now.
Step 1-OK-Scaffold a SAPUI5 project (yo easy-ui5)
Step 2-OK-Inspect the generated code (cd com.myorg.myUI5App/ code .)
Step 3-OK-Test the web-app locally (npm start)
Step 4-OK-Package the MTA archive (npm run build:mta)

3.
I executed g [npm i mbt](url). And here is the result :
![image](https://user-images.githubusercontent.com/23510132/72572921-5e983400-38c4-11ea-9444-cc637564e7cb.png)
 
PS :

**Comment by IObert** [comment]
Thanks for running the tests, I'm glad to see that it worked in scenario 2. 
I suspect the first one failed because of missing Windows file access rights but my French is too bad to actually understand the error message 😕

I just tested the tutorial in a brand new Windows environment (Azure VM) with PowerShell and I worked there. So it doesn't seem to be an issue with the tutorial or the generator. I rather suspect that the `mbt` tool has a bug.

I found several people reporting similar issues [here](https://github.com/SAP/cloud-mta-build-tool/issues?utf8=%E2%9C%93&q=could+not+execute+the+make) . Some of them say it works from the "git terminal" but not from the Windows terminal.
I'm afraid I have to ask you one more time to open a new issue at that project, sorry.

PS: You can write `SAPDocuments/Tutorials#4490` to reference this issue in a [new one on the other project](https://github.com/SAP/cloud-mta-build-tool/issues/new/choose).
